dev-support/create-release/do-release-docker.sh

```diff
@@ -85,12 +86,14 @@ IMGTAG=latest
 JAVA=
 RELEASE_STEP=
 GIT_REPO=
-while getopts "d:fhj:p:r:s:t:" opt; do
+MAVEN_VOLUME=
+while getopts "d:fhj:m:p:r:s:t:" opt; do
   case $opt in
     d) WORKDIR="$OPTARG" ;;
     f) DRY_RUN=0 ;;
     t) IMGTAG="$OPTARG" ;;
     j) JAVA="$OPTARG" ;;
+    m) MAVEN_VOLUME="$OPTARG" ;;
```

Comment:
why not lake this a flag, and just always use a path under the build directory? You want to reuse it across invocations of do-release? That strikes me as undesirable.

Reply:
Named volumes have different properties then bind mounts like the build directory.

I specifically want to reuse across RCs because dependencies do not usually change between RCs for a given release line.